public/views/pages/payments/stripe/checkout_session_completed_webhook.liquid

```diff
@@ -14,8 +14,12 @@ layout: null
     if object.valid
       echo object
     else
-      log object, type: 'ERROR: payments_stripe/checkout_session_completed_webhook object'
-      response_status 500
+      # Stripe will ping all endpoints so maybe we get another instance's ID
+      # and that's why the command will not valid.
+      # We should not send 500 status response in this case, because the request was valid,
+      # but it was not relevant in the current instance.
+      log object, type: 'WARNING: payments_stripe/checkout_session_completed_webhook object'
+      response_status 202
```

Comment:
@hosszukalman Is the ping happening on initialization? Can we detect ping request and handle it properly?

I'm not sure about always returning 200, we will not be able to track errors from stripe dashboard. I guess they encourage developers to use proper response codes as they implemented retry mechanism https://stripe.com/docs/webhooks#built-in-retries

Reply:
@piotrze they ping all created webhooks on the account and as we won't handle other any instance transaction IDs they wrote an email that they don't want to retry to send the data again. We can check if the error is on ID field and if not, we can set error status code if it's sounds better.

Reply:
I think the problem is that we are using same token for different instances. 
On staging we might want to check if webhook is related with particular instance(probably check the transaction id), so we don't have to create separate accounts on stripe for developers.

Production sites has separate stripe accounts. For production we should return 500 if webhook is not for this particular instance.

Reply:
Our client uses one stripe account for several production sites and I think this is a real use case.

Reply:
yes, DCRA will use same stripe account for multiple applications. What I would do here is to check whether we have related `checkout_session` in the db, if not then `response_status 202`, if we have it and `!object.valid` then we should return 500

Reply:
Yes I agree with you @streflik , I also suggested it here `We can check if the error is on ID field and if not, we can set error status code if it's sounds better.` It means that if the transaction ID is not found in the database, we can set `202` otherwise we can response with an error.
@piotrze ?

Reply:
Ok, but it feels strange that they have one account for multiple sites.

```
You must use separate Stripe accounts for projects, websites, or businesses that operate independently from one another. When you activate a new account, it’s subject to Stripe’s standard policies and pricing—it doesn’t inherit any special status or other similar considerations that might apply to your existing account.
```
https://stripe.com/docs/account/multiple-accounts

Reply:
Definitely we need to differentiate situation when webhook comes with transactionId that is not in db from situation when we have application error.

Reply:
I just updated the code @piotrze, @streflik. Could you have a look?